doc/gui/examples/matplotlib_example.py

```diff
@@ -0,0 +1,22 @@
+# Integrating Matplotlib into Taipy using figure property

```

Comment:
Where are you using the figure property?

Reply:
Sir, I am using matplotlib.figure property as this class is automatically recognized by Taipy GUI and can be passed to the content property. Thus the Matplotlib output can be inserted in the part block using figure property.

Reply:
This is a bit confusing as Taipy has a figure property for its charts. What about: # Integrating Matplotlib Figure into Taipy

Reply:
It is confusing indeed and I plan to fix that when I'm done with (far) more urgent tasks.
This really is a demo of how to use the part block to hold a Matplotlib figure. That's where it will go, and the chart control documentation will reference that.

Reply:
I am fine with that. I am confused by the wording above and propose one that is less confusing in my opinion

Reply:
I understand sir. Have removed "# Integrating Matplotlib Figure into Taipy" for now. If there can be a comment which depicts it better, we can go ahead with that sir. Thank you sir.